src/index.d.ts

```diff
@@ -491,13 +491,12 @@ declare namespace admin.auth {
     /**
      * The date the user last signed in, formatted as a UTC string.
      */
-    lastSignInTime: string;
+    lastSignInTime?: string | null;
```

Comment:
nb: technically, a public api change. Arguably breaking since typscript users may have to adjust their code. (Note that we've *always* allowed this to be null... we've just not defined it as such. IOW, I think this is a bugfix more than a change.)

Reply:
I think the `?` is more of a significant change. Can we remove that?